src/styles/index.scss

```diff
@@ -1,3 +1,68 @@
+@import 'variables';
+
 body {
   margin: 0;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: grid;
+  grid-template-columns: repeat(7, $day-size);
+  gap: $day-gap;
+  box-sizing: border-box;
+  padding: $calendar-padding;
+  width: calc(
+    (7 * $day-size) + (6 * $day-gap) + (2 * $calendar-padding)
+  );
+}
+
+.calendar__day {
+  position: relative;
+  width: $day-size;
+  height: $day-size;
+  box-sizing: border-box;
+  border: $day-border;
+  background-color: $day-color;
+
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &::before {
+    content: '';
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 30px;
+  }
+
+  &:hover {
+    cursor: pointer;
+    background-color: $day-hover-color;
+    transform: translateY(-20px);
+    transition:
+      background-color 0.5s,
+      transform 0.5s;
+  }
```

Comment:
It's recommended to put `transition` property in the original class, not in the `:hover` state. The reason is that `transition` in the original state will cover both `:hover on` and `:hover off`, but if it's in the `:hover` state, it will not cover `:hover off`. So, the transition effect will not be smooth when you move the mouse out.